Easydict.xcodeproj/project.pbxproj

```diff
@@ -3251,6 +3311,14 @@
 				minimumVersion = 5.8.1;
 			};
 		};
+		EA3B81FA2B52555C004C0E8B /* XCRemoteSwiftPackageReference "Defaults" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/sindresorhus/Defaults.git";
+			requirement = {
+				branch = main;
+				kind = branch;
```

Comment:
Why does `Defaults` use branches instead of version numbers?

<img width="1102" alt="image" src="https://github.com/tisfeng/Easydict/assets/25194972/4cb1ac4d-9354-4632-9c3e-adae8e79e57d">

Reply:
fixed.